assets/stylesheets/responsive/custom.scss

```diff
@@ -445,7 +445,9 @@ color: $righttoknow-heading-color;
 }
 
 .make-request-quick-button {
-  margin-top: 1em;
+    max-width: 11em;
```

Comment:
This looks like it uses different indentation and an extra line of whitespace compared to the rest of this file. Could you please fix add a commit to correct that @Keerti-Gautam